docs/bundles/SyliusTaxationBundle/custom_use_of_bundle_out_side_sylius.rst

```diff
@@ -0,0 +1,329 @@
+            >
+                <join-column name="tax_category_id" referenced-column-name="id" nullable="false" />
+            </many-to-one>
+        </entity>
+
+    </doctrine-mapping>
+
+ Updating database schema again.
+
+.. code-block:: bash
+
+php app/console doctrine:schema:update --force
+
+Done! Now your **Guest** model can be used in Sylius taxation engine.
+
+Forms
+~~~~~
+
+If you want to add a tax category selection field to your model form, simply use the ``sylius_tax_category_choice`` type.
+
+.. code-block:: php
+
+    namespace AcmeBundle\Form\Type;
+
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\Form\AbstractType;
+
+    class ServerType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder
+                ->add('name', 'text')
+                ->add('dob', 'text')
+                ->add('taxCategory', 'sylius_tax_category_choice')
+            ;
+        }
+
+        public function getName()
+        {
+            return 'acme_guest';
+        }
+    }
+
+
+Configuring taxation
+--------------------
+
+To start calculating taxes, we need to configure the system. In most cases, the configuration process is done via web interface, but you can also do it programatically.
+
+Creating the tax categories
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First step is to create a new tax category.
+
+.. code-block:: php
+
+    <?php
+
+    public function configureCategoryAction()
+    {
+        $repository = $this->container->get('sylius.repository.tax_category');
+        $manager = $this->container->get('sylius.manager.tax_category');
+
+        $guestTaxCategory = $repository
+            ->createNew()
+            ->setName('Guest')
+            ->setDescription('Visitors to guesthouse/hostel/hotel')
+        ;
+
+        $manager->persist($guestTaxCategory);
+
+        $manager->flush();
+    }
+
+Categorizing the taxables
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Second thing to do is to classify the taxables, in our example we'll get three guests and assign the proper categories to them.
+
+.. code-block:: php
+
+    <?php
+
+    public function configureTaxableAction()
+    {
+        $guestAdult = // ... Guest with age 18+
+        $guestMinor = // ... Guest that is 12 - 18 years old
+        $guestChild = // ... Guest is child
+
+        $repository = $this->container->get('sylius.repository.tax_category');
+        $guestTaxCategory = $repository->findOneBy(array('name' => 'Guest'));
+
+        $guestAdult->setTaxCategory($guestTaxCategory);
+        $guestMinor->setTaxCategory($guestTaxCategory);
+        $guestChild->setTaxCategory($guestTaxCategory);
+
+        // ...
+
+        // Save the product entities.
+        $manager->persist($guestAdult);
+        $manager->persist($guestMinor);
+        $manager->persist($guestChild);
+
+    }
+
+Configure calculator
```

Comment:
Changes should be done here -> http://docs.sylius.org/en/latest/bundles/SyliusTaxationBundle/custom_calculators.html